src/graphics/vulkan.zig

```diff
@@ -195,6 +206,11 @@ const deviceExtensions = [_][*:0]const u8{
 	c.VK_KHR_SWAPCHAIN_EXTENSION_NAME,
 };
 
+const deviceFeatures: c.VkPhysicalDeviceFeatures = .{
```

Comment:
Okay, so I was avoiding pointing it out before because there are no technical requirements known to me which would require you to use features not listed in `VkPhysicalDeviceFeatures`, but for sake of documentation:
As you probably noticed `VkPhysicalDeviceFeatures` does not contain `pNext` entry. This means it cannot be extended with additional structures and therefore with new features, added after Vk1.0. If it happens later that you will need any of those features like [those from VK1.2](https://registry.khronos.org/vulkan/specs/latest/man/html/VkPhysicalDeviceVulkan12Features.html) we will have to change this code significantly, thus I would avoid improving it too much.

Reply:
Would that require any more work besides replacing the struct type and calling the Vulkan 1.2 function?

Reply:
Yes, there is no Vulkan 1.2 function.
Instead you use `VkPhysicalDeviceFeatures2` struct and `vkGetPhysicalDeviceFeatures2` (Added in vk 1.1 i think) and chain [`VkPhysicalDeviceVulkan12Features`](https://registry.khronos.org/vulkan/specs/latest/man/html/VkPhysicalDeviceVulkan12Features.html) + friends to `VkPhysicalDeviceFeatures2` pNext pointer.
I might have exaugurated the changes required for that to work tbh, it would be more of a extension of that system.